static/app/actionCreators/pageFilters.tsx

```diff
@@ -70,6 +83,10 @@ type PageFiltersUpdate = {
   utc?: boolean | null;
 };
 
+type CodecovPageFiltersUpdate = {
```

Comment:
Why do we need a distinction of Codecov vs Sentry? If we are merging Codecov to Sentry, then this should not exist, and it should just be Sentry going forward. 

I'm not a fan of this approach, as I'm fairly sure it will linger around until someone ultimately needs to address it (at which point, we'll have a lot of the code relying on this and it'll be a massive PITA to update)